server/src/main/java/dev/slimevr/tracking/trackers/udp/TrackersUDPServer.java

```diff
@@ -331,16 +340,14 @@ public void run() {
 							} else {
 								conn.timedOut = false;
 								for (Tracker value : conn.getTrackers().values()) {
-									IMUTracker tracker = (IMUTracker) value;
-									if (tracker.getStatus() == TrackerStatus.DISCONNECTED)
-										tracker.setStatus(TrackerStatus.OK);
+									value.setStatus(TrackerStatus.OK);
```

Comment:
this will reset error state, should only set to OK if it was DISCONNECTED previously